TvMaterialCatalog/app/src/main/java/com/google/tv/material/catalog/HomeGridCard.kt

```diff
@@ -7,26 +7,29 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.unit.dp
+import androidx.tv.material3.Card
 import androidx.tv.material3.CardDefaults
-import androidx.tv.material3.CardLayoutDefaults
-import androidx.tv.material3.ExperimentalTvMaterial3Api
-import androidx.tv.material3.StandardCardLayout
+import androidx.tv.material3.StandardCardContainer
 import androidx.tv.material3.Text
 
-@OptIn(ExperimentalTvMaterial3Api::class)
 @Composable
 fun ComponentsGridCard(
     component: Component,
-    modifier: Modifier = Modifier
+    modifier: Modifier = Modifier,
+    onClick: () -> Any = {}
 ) {
     val image = getHomeGridCardImage(imageArg = component.imageArg)
     val navHostController = LocalNavController.current
 
-    StandardCardLayout(
+    StandardCardContainer(
         modifier = modifier,
         imageCard = {
-            CardLayoutDefaults.ImageCard(
-                onClick = { navHostController.navigate(component.routeValue) },
+            Card(
+                onClick = {
```

Comment:
The code to handle click events should be updated as follows:

```
Card(
   onClick = {
     onClick()
     navHostController.navigate(component.routeValue)
  }
```

Reply:
I've changed this such that the click handler is provided into ComponentsGridCard, ptal.